README.md

```diff
@@ -97,13 +104,15 @@ This will load the various `.env` files for Algorand, Circle, etc. credentials -
 most other environment variables will be overridden in favor of those specified
 in the `docker-compose.yml` file.
 
-Next, the CMS key needs to be added to the admin user created by Directus.
+### Adding the CMS key to the admin user
+
+The CMS key needs to be added to the admin user created by Directus.
```

Comment:
This was embedded in the "docker compose" section but it's relevant no matter how a developer spins up a local environment.